test/suites/E2E/test_uc_performance.py

```diff
@@ -11,160 +12,83 @@
 )
 from common.uc_eval.utils.data_class import ModelConfig, PerfConfig
 
+perf_scenarios = [
+    # (mean_in, mean_out, max_req, concurrent, sampling, hit_rate)
+    (4000, 1024, 1, 1, "{}", 80),
+    (4000, 1024, 8, 8, "{}", 80),
+]
+scenario_ids = [f"in_{s[0]}-out_{s[1]}-con_{s[3]}" for s in perf_scenarios]
```

Comment:
```
scenario_ids = [f"in_{s[0]}-out_{s[1]}-con_{s[3]}" for s in perf_scenarios]
```
maybe we can hide this param from user and generate a unique id for each case automatically